src/tpm2_tis.c

```diff
@@ -57,8 +57,13 @@ enum tpm_tis_int_flags {
     TPM_INTF_DATA_AVAIL_INT     = 0x001,
 };
 
-
+#ifndef TPM_BASE_ADDRESS
+#ifdef WOLFTPM_MMIO_BUILTIN_CB
```

Comment:
I think this define could simply be WOLFTPM_MMIO

Reply:
I think that `WOLFTPM_MMIO_BUILTIN_CB` clarify better that custom I/O callbacks cannot be used together with this feature.